src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.ts

```diff
@@ -36,6 +38,20 @@ export const addAggregatorAndDestinationToMemo = ({
     'expected finalAssetLimitWithManualSlippage to be a positive amount',
   )
 
+  const minimumPrecision = 5
+  const endingExponential = finalAssetPrecision - minimumPrecision
+
+  // The uniswap aggregator expects this amount to be an exponent, we need to add two numbers at the end which are used at exponents in the contract
```

Comment:
i.e all TC aggregators
```suggestion
  // The THORChain aggregators expects this amount to be an exponent, we need to add two numbers at the end which are used at exponents in the contract
```